00_hello_world/main.go

```diff
@@ -9,5 +9,20 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var firstNumber = 1
```

Comment:
This is perfectly valid, but it's more normal in modern maths to start the fibonacci sequence at 0.

Reply:
Done.